packages/commonwealth/client/scripts/views/pages/view_thread/ViewThreadPage.tsx

```diff
@@ -130,6 +137,15 @@ const ViewThreadPage = ({ identifier }: ViewThreadPageProps) => {
     thread?.topic?.id,
   );
 
+  //TODO:
```

Comment:
remove todo, please make sure you take a look on your PR before you ask others, so you can catch those kind of things